algorithms/refinement/reflection_manager.py

```diff
@@ -72,16 +73,20 @@
       .type = float(value_min = 0)
       .expert_level = 1
 
-    trim_scan_edges = 0.0
+    scan_margin = 0.0
       .help = "Reflections within this value in degrees from the centre of the"
               "first or last image of the scan will be removed before"
               "refinement, unless doing so would result in too few remaining"
               "reflections. Reflections that are truncated at the scan edges"
               "have poorly-determined centroids and can bias the refined model"
               "if they are included."
-      .type = float(value_min=0,value_max=1)
+      .type = float(value_min=0,value_max=5)
       .expert_level = 1
 
+    trim_scan_edges = 0.0
```

Comment:
```suggestion
    trim_scan_edges = None
```
This makes the check for deprecation purposes more reliable, and makes
```dials.command scan_margin=5.0 trim_scan_edges=0.0``` behave similarly to
```dials.command scan_margin=5.0 trim_scan_edges=0.00000001``` instead of entirely different